deployments/deployment.yaml

```diff
@@ -35,6 +35,16 @@ objects:
             value: "${JWT_MODULE}"
           - name: KEYCLOAK_SERVER
             value: "${KEYCLOAK_SCHEME}://${KEYCLOAK_HOST}:${KEYCLOAK_PORT}${KEYCLOAK_PATH}"
+          - name: AWS_CLIENT_ID
```

Comment:
Since we're now dealing with being able to setup multiple providers per endpoint, maybe it's worth having a provider-specific names here? Something like `COGNITO_APP_CLIENT_ID` that way we don't conflate things with other AWS services we have like SES here: https://github.com/RedHatInsights/mbop/pull/31/files#diff-4f049da0ef258bd66374745e0c2f5d7df11a2ce912c32e100b2f7ef4485afdfbR38

Reply:
Sounds great!  I'll change it to that!